R/ts_gtrends_windows.R

```diff
@@ -40,18 +68,14 @@ ts_gtrends_windows <- function(keyword = NA,
     }) %>%
     # Make sure the last window is large enough to yield the desired frequency
     rowwise() %>%
-    mutate(start_date = ifelse(prevent_window_shrinkage & end_date == Sys.Date(), seq(start_date, length.out = 2, by = paste0("-", windowsize))[1], start_date)) %>%
+    mutate(start_date = ifelse(prevent_window_shrinkage & end_date == Sys.Date(),
```

Comment:
just cosmetics, but can you do: https://style.tidyverse.org/syntax.html#long-lines

Reply:
Good to know, thanks. Adjusted layout in 79874f6